src/broker/modservice.c

```diff
@@ -72,54 +72,74 @@ static modservice_ctx_t *getctx (flux_t *h, module_t *p)
     return ctx;
 }
 
-static void stats_get_cb (flux_t *h, flux_msg_handler_t *mh,
-                          const flux_msg_t *msg, void *arg)
+static void stats_get_cb (flux_t *h,
+                          flux_msg_handler_t *mh,
+                          const flux_msg_t *msg,
+                          void *arg)
 {
     flux_msgcounters_t mcs;
 
     flux_get_msgcounters (h, &mcs);
 
-    if (flux_respond_pack (h, msg, "{ s:i s:i s:i s:i s:i s:i s:i s:i }",
-                           "#request (tx)", mcs.request_tx,
-                           "#request (rx)", mcs.request_rx,
-                           "#response (tx)", mcs.response_tx,
-                           "#response (rx)", mcs.response_rx,
-                           "#event (tx)", mcs.event_tx,
-                           "#event (rx)", mcs.event_rx,
-                           "#keepalive (tx)", mcs.keepalive_tx,
-                           "#keepalive (rx)", mcs.keepalive_rx) < 0)
-      FLUX_LOG_ERROR (h);
+    if (flux_respond_pack (h,
+                           msg,
+                           "{ s:i s:i s:i s:i s:i s:i s:i s:i }",
+                           "#request (tx)",
+                           mcs.request_tx,
+                           "#request (rx)",
+                           mcs.request_rx,
+                           "#response (tx)",
+                           mcs.response_tx,
+                           "#response (rx)",
+                           mcs.response_rx,
+                           "#event (tx)",
+                           mcs.event_tx,
+                           "#event (rx)",
+                           mcs.event_rx,
+                           "#keepalive (tx)",
+                           mcs.keepalive_tx,
+                           "#keepalive (rx)",
+                           mcs.keepalive_rx)
```

Comment:
:scream:

Reply:
So, I was setting up clang-format for a new project today, and for some reason this came to mind with a semi-solution.  I notice that in flux, we actually bin-pack arguments and parameters all over the place.  When I wrote the config I thought we did not do that (I normally try to avoid it so that's what I put in)  but with bin-packing turned on, slightly different results are possible.  Here's a long json_pack with the kind of settings we had in this PR originally:

```c
  return json_pack("{ s:f, s:f, s:f, s:I, s:I, s:I, s:I, s:I, s:I }",
                   "ctime",
                   stats->ctime,
                   "starttime",
                   stats->starttime,
                   "lastrun",
                   stats->lastrun,
                   "count",
                   stats->count,
                   "failcount",
                   stats->failcount,
                   "total",
                   stats->total,
                   "success",
                   stats->success,
                   "failure",
                   stats->failure,
                   "deferred",
                   stats->deferred);
```

Playing with bin-packing though, here are some options:

```c
// Just raw, similarly crappy if not worse, but denser
  return json_pack("{ s:f, s:f, s:f, s:I, s:I, s:I, s:I, s:I, s:I }", "ctime",
                   stats->ctime, "starttime", stats->starttime, "lastrun",
                   stats->lastrun, "count", stats->count, "failcount",
                   stats->failcount, "total", stats->total, "success",
                   stats->success, "failure", stats->failure, "deferred",
                   stats->deferred);

// ending empty comment after prefix arguments, now they're paired and packed, kinda like
  return json_pack("{ s:f, s:f, s:f, s:I, s:I, s:I, s:I, s:I, s:I }",  //
                   "ctime", stats->ctime, "starttime", stats->starttime,
                   "lastrun", stats->lastrun, "count", stats->count,
                   "failcount", stats->failcount, "total", stats->total,
                   "success", stats->success, "failure", stats->failure,
                   "deferred", stats->deferred);

// ending empty comments after each pair, clang-format now gives us what we wanted.
  return json_pack("{ s:f, s:f, s:f, s:I, s:I, s:I, s:I, s:I, s:I }",  //
                   "ctime", stats->ctime,                              //
                   "starttime", stats->starttime,                      //
                   "lastrun", stats->lastrun,                          //
                   "count", stats->count,                              //
                   "failcount", stats->failcount,                      //
                   "total", stats->total,                              //
                   "success", stats->success,                          //
                   "failure", stats->failure,                          //
                   "deferred", stats->deferred);
```

Thoughts?